session/src/test/java/uk/gov/di/ipv/cri/common/api/service/SignedJWTBuilder.java

```diff
@@ -25,9 +25,10 @@
 import java.time.temporal.ChronoUnit;
 import java.util.Date;
 import java.util.Map;
+import java.util.Objects;
 import java.util.UUID;
 
-class SignedJWTBuilder {
+class SignedJWTBuilder<T> {
```

Comment:
What's the reasoning behind using a generic type here?

Reply:
Yeah premature optimisation on my part was thinking of the general use of SignedJWTBuilder`PersonIdentityClaim` or `AddressClaim` I suppose I could have just made `Object` but given we are moving away from Java we are unlikely to use this in future